pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -794,6 +817,14 @@ func (m *kubeGenericRuntimeManager) doPodResizeAction(ctx context.Context, pod *
 			logger.Error(err, "Failed to notify runtime for UpdatePodSandboxResources", "resource", rName, "pod", klog.KObj(pod))
 			// Don't propagate the error since the updatePodSandboxResources call is best-effort.
 		}
+
+		if utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodLevelResourcesVerticalScaling) {
+			if err = m.actuatedState.SetPodLevelResources(pod.UID, pod.Spec.Resources); err != nil {
```

Comment:
You also need to initialize the actuated state when the pod is first created. Here is where we currently initialize the pod cgroup: https://github.com/kubernetes/kubernetes/blob/3ec72e2e973664a0690bc7122c9cd291702b5f36/pkg/kubelet/kubelet.go#L2064

I guess that would be the best place to call it?